### PR TITLE
Amending audio/preview2 to use different headers

### PR DESCRIPTION
**Pull request by davet1985**
### JIRA link (if applicable) ###

N/A

### Change description ###

Trying with non-audio header to see if that's causing some issues.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by richardbull — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java [hunk 1]
```diff
@@ -88,14 +88,14 @@ public ResponseEntity<Resource> preview(Integer mediaId) {
     public ResponseEntity<byte[]> preview2(Integer mediaId, @RequestHeader(value = "Range", required = false) String httpRangeList) {
         InputStream audioMediaFile = audioService.preview(mediaId);
 
+        byte[] bytes = IOUtils.toByteArray(audioMediaFile);
         if (httpRangeList == null) {
-            byte[] bytes = IOUtils.toByteArray(audioMediaFile);
             return ResponseEntity.status(HttpStatus.OK)
-                .header("Content-Type", "audio/mpeg")
+                .header("Content-Type", "application/octet-stream")
+                .header("Content-Disposition", "attachment; filename=\"" + mediaId.toString() + ".mp3\"")
```
Comment (richardbull) [inline src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java:95]: I think this header will cause the browser to handle a file download - as this is what I'm using for the download transcript endpoint

File: test.mp3
Comment (richardbull) [inline test.mp3:1]: I'm guessing you didn't mean to commit this file as you have it under `src/main/resources/test.mp3`?

File: src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java (hunk 1 above)
Comment (davet1985) [reply src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java:95]: Yeah, I expect so, I'm just trying a different header to see if that's what's causing the request to be blocked.

File: test.mp3
Comment (davet1985) [reply test.mp3:1]: nope, removed

**Pull request merged**